apps/api/src/database/database.module.ts

```diff
@@ -50,7 +47,8 @@ const environmentSpecificConfig = getEnvironmentSpecificConfig(nodeEnv);
 const appOrmConfig: PostgresConnectionOptions = {
   ...config,
   ...environmentSpecificConfig,
-  migrationsRun: false,
+  migrationsRun: true,
+  synchronize: false,
```

Comment:
Had to change these to:
migrationsRun: false,
synchronize: true

Otherwise everything else is working as expected. Planning_Session ID -> Occupations/Activities gets saved in the database.